src/hooks/use-timer.ts

```diff
@@ -14,7 +14,7 @@ export const useTimer = ({
   end,
   updateInterval,
 }: UseTimerParams) => {
-  const intervalId = useRef<NodeJS.Timer>();
+  const intervalId = useRef(0 as unknown as NodeJS.Timeout);
```

Comment:
src/hooks/use-timer.ts:43:32 - error TS2769: No overload matches this call.
  Overload 1 of 3, '(intervalId: string | number | Timeout | undefined): void', gave the following error.
    Argument of type 'Timer | undefined' is not assignable to parameter of type 'string | number | Timeout | undefined'.
      Type 'Timer' is not assignable to type 'string | number | Timeout | undefined'.
        Property '[Symbol.dispose]' is missing in type 'Timer' but required in type 'Timeout'.
  Overload 2 of 3, '(handle: number): void', gave the following error.
    Argument of type 'Timer | undefined' is not assignable to parameter of type 'number'.
      Type 'undefined' is not assignable to type 'number'.
  Overload 3 of 3, '(handle: number): void', gave the following error.
    Argument of type 'Timer | undefined' is not assignable to parameter of type 'number'.
      Type 'undefined' is not assignable to type 'number'.

43     return () => clearInterval(intervalId.current);

Reply:
ReturnType<typeof setTimeout>